README.md

```diff
@@ -1,212 +1,57 @@
-**Will only work in the cluster node**
-#### Pre-requisites:
-1. Tekton pipelines & dashboard installed
-2. dashboard repo cloned
-
-#### Steps:
-1. Edit `ingress/ingress-https-setup.sh` with all the necessary info
-2. Run the script from within the dashboard repo
-3. Access dashboard via `https://tekton-dashboard.<IP_ADDRESS>.nip.io`
-
-
-## Install on Minishift
-
-Either follow the instructions for OpenShift above or use the operator install as per the instructions below.
-
-1. Install [tektoncd-pipeline-operator](https://github.com/openshift/tektoncd-pipeline-operator#deploy-openshift-pipelines-operator-on-minikube-for-testing)
-2. [Checkout](https://github.com/tektoncd/dashboard/blob/master/DEVELOPMENT.md#checkout-your-fork) the repository
-
-If you want to install the Dashboard into the tekton-pipelines namespace:
-
-- Install the Dashboard `./minishift-install-dashboard.sh`
-
-If you want to install the Dashboard into any other namespace:
-
-- Install the Dashboard `./minishift-install-dashboard.sh -n {NAMESPACE}`
-
-3. Wait until the pod `tekton-dashboard-1` is running in the namespace the Dashboard is installed into
-
-## Accessing the Dashboard
+![Dashboard UI workloads page](docs/dashboard-ui.png)
 
-The Dashboard can be accessed through its ClusterIP Service by running `kubectl proxy`. Assuming tekton-pipelines is the install namespace for the Dashboard, you can access the web UI at `localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/`.
+## Start using Tekton Dashboard
 
-An alternative way to access the Dashboard is using `kubectl port-forward` e.g. if you installed the Tekton Dashboard into the `tekton-pipelines` namespace (which is the default) you can access the Dashboard with `kubectl --namespace tekton-pipelines port-forward svc/tekton-dashboard 9097:9097` and then just open `localhost:9097`.
+- Tekton Dashboard [install](./docs/install.md)
+- Tekton Dashboard [extensions](./docs/samples/extension/README.md)
+- Try out the [katacoda tutorial](https://katacoda.com/ncskier/scenarios/tekton-dashboard)
+- Take a look at our [roadmap](./roadmap.md)
+- Feature requests and bug reports welcome, please [open an issue](https://github.com/tektoncd/dashboard/issues/new/choose)
 
-## Accessing the Dashboard on Minishift
+## Read the docs
 
-The Dashboard can be accessed by running `kubectl --namespace tekton-pipelines port-forward svc/tekton-dashboard 9097:9097`
-If installed into a namespace other than tekton-pipelines then the Dashboard can be accessed by running `kubectl --namespace $NAMESPACE port-forward svc/tekton-dashboard 9097:9097`
-You can access the web UI at `http://localhost:9097/`
+It is **strongly recommended** to use the **v0.6.1.4** release for Tekton Pipelines v0.11.x and v0.12.x and Tekton Triggers v0.4
+- This is a **critically important security release**
+- The following versions are deprecated and should be used for **development or isolated usage only**
```

Comment:
```suggestion
- Earlier versions are deprecated and should be used for **development or isolated usage only**
```

The table below includes newer releases and master, so saying 'the following versions' isn't correct.